pyccl/pyutils.py

```diff
@@ -522,6 +522,45 @@ def _fftlog_transform(rs, frs,
     return ks, fks
 
 
+def _fftlog_transform_general(
+    rs, frs, mu, q, spherical_bessel, bessel_deriv, window_frac
+):
```

Comment:
Can we add a comment (doesn't have to be a proper docstring) here saying a bit what the expected inputs and outputs are? Nothing long, just so developers in the future can see what this does and perhaps reuse it for other calculations.